gammapy/obs/observation.py

```diff
@@ -85,3 +91,244 @@ def select_linspace_subset(self, num):
         # Round down to nearest integer
         indices = indices.astype('int')
         return self[indices]
+
+    def select_obs_var_range(self, selection_variable,
```

Comment:
I find `select_obs_var_range` is prettly long and cryptic. How about `select_minmax` or `select_range`?